src/components/status-panel/style.css

```diff
@@ -0,0 +1,6 @@
+.status {
```

Comment:
File is a duplicate of the scss style, looks like it can be removed.

Reply:
👌 e46b181